File: Tugas coding modul 2/coding c/PRAK203-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){
    float a, b, i, j, x, y, hasil;

    // 20 3 4 12 5 9 | 12 2 10 4 3 14

    printf("Input\n");
    scanf("%f %f %f %f %f %f", &a, &b, &i, &j, &x, &y);

    hasil = (float) (a - b) * i / j - (x + y);

    printf("\nOutput\n");
    printf("Hasil = %.3f", hasil);
}

File: Tugas coding modul 2/coding py/PRAK203-2310817120005-Noviana_Nur_Aisyah.py
# 20 3 4 12 5 9 | 12 2 10 4 3 14
print('Input')
input_aja = input()

angka = input_aja.split()

a, b, i, j, x, y = map(float, angka)

# a dikurang b dikali dengan i dibagi j dikurang dengan x ditambah y 
hasil = (a - b) * i / j - (x + y)
print('\nOutput')
print(f'{hasil:.3f}')